datetime/datetime.go

```diff
@@ -283,7 +289,17 @@ func (tm *Datetime) UnmarshalMsgpack(b []byte) error {
 			}
 			zone = indexToTimezone[int(dt.tzIndex)]
 		}
-		loc = time.FixedZone(zone, offset)
+		if zone != NoTimezone {
+			if loadLoc, err := time.LoadLocation(zone); err == nil {
+				loc = loadLoc
+			} else {
+				// Unable to load location.
```

Comment:
Since we are a library it does not looks like a good idea to throw a warning message on each datetime with a concreate timezone ('MSK').

Reply:
Well, ok, let it be like it. Are our conversion rules displayed in docs?

Reply:
That's a good point, I added a comment:
```patch
diff --git a/datetime/datetime.go b/datetime/datetime.go
index 4eb23bb..0fe20f5 100644
--- a/datetime/datetime.go
+++ b/datetime/datetime.go
@@ -224,6 +224,13 @@ func (dtime *Datetime) Interval(next *Datetime) Interval {
 }
 
 // ToTime returns a time.Time that Datetime contains.
+//
+// If a Datetime created from time.Time value then an original location is used
+// for the time value.
+//
+// If a Datetime created via unmarshaling Tarantool's datetime then we try to
+// create a location with time.LoadLocation() first. In case of failure, we use
+// a location created with time.FixedZone().
```

Reply:
What happens if we create a `Datetime` from `time.Time` with unknown timezone (like `Local`)? The error is raised?

Reply:
Yes, it is:

https://github.com/tarantool/go-tarantool/blob/86390b6b3337962a395fc5744aec124d41d2af4b/datetime/datetime.go#L100-L101
https://github.com/tarantool/go-tarantool/blob/86390b6b3337962a395fc5744aec124d41d2af4b/datetime/datetime.go#L113-L114